cypress/integration/page.spec.js

```diff
@@ -949,23 +1027,23 @@
     });
 
     it('should allow to enter an author name', () => {
-      newCommentForm.nameInput().type('Some name')
+      newCommentForm.nameInput().type('Some name');
       newCommentForm.nameInput().should('have.value', 'Some name');
     });
 
     it('should allow to enter an author email', () => {
-      newCommentForm.emailInput().type('some@email.com')
+      newCommentForm.emailInput().type('some@email.com');
       newCommentForm.emailInput().should('have.value', 'some@email.com');
     });
 
     it('should allow to enter a comment body', () => {
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
Consider using a more descriptive comment body for the test input to improve readability.